oauth2client/contrib/django_util/__init__.py

```diff
@@ -276,7 +382,7 @@ def __init__(self, request, scopes=None, return_url=None):
 
         # make sure previously requested custom scopes are maintained
         # in future authorizations
-        credentials = storage.get_storage(self.request).get()
+        credentials = self.credentials
```

Comment:
The purpose of 1) is to avoid the inheritance problem discussed in Item 17 of _Effective Java_ (Second Edition) and to avoid the abstraction problem discussed [here](https://www.youtube.com/watch?v=AFhSWQYdjy4&t=17m50s).